crates/ruma-events/src/ignored_user_list.rs

```diff
@@ -68,12 +68,12 @@ mod tests {
         });
 
         assert_matches!(
-            from_json_value::<Raw<BasicEvent<AnyBasicEventContent>>>(json)
+            from_json_value::<Raw<GlobalAccountDataEvent<AnyGlobalAccountDataEventContent>>>(json)
```

Comment:
This test seems to be outdated. Do you mind rewriting it to use `Raw<AnyGlobalAccountDataEvent>` over the current `Raw<_>` type?

Reply:
I'm not sure I understand what you mean

Reply:
I mean

```suggestion
            from_json_value::<Raw<AnyGlobalAccountDataEvent>>(json)
```

here and below change `GlobalAccountDataEvent { content: AnyGlobalAccountDataEventContent::IgnoredUserList::… }` to `AnyGlobalAccountDataEvent::IgnoredUserList(…)`.